app-backend/migrations/src_migrations/main/scala/63.scala

```diff
@@ -0,0 +1,77 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M63 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(63)(List(
+// Note:
+// The -> operator extracts the JSON value at a given key
+// The ->> operator extracts the JSON value at a given key *as a string*
+// Given a JSON field like '{"a": null}'
```

Comment:
Yes, oops! I was explaining some counterintuitive behavior I encountered but apparently got distracted. Will update so you can find out the ending.